### DISCUSSION TITLE
Welcome to halstack-angular Discussions!

### DISCUSSION THREAD

**Discussion by peibolsang** [dxc-technology/halstack-angular#176] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by meghananatorahathula** [comment]
Hi Team,
 I am Meghana. I am a UI developer in dxc .I am using dxc angular cdk components. version 4.I am facing one issue with drop-down(dxc-select) selection panel in case of table scroll. I want a feature to make the drop-down selection panel outside of the scrolling table.  I tried but, I didn't find any solution.
![issue_dropdown](https://github.com/dxc-technology/halstack-angular/assets/86658556/4d0c8e84-73f7-496a-8d97-81b0788a538e)
could you help me to find any solution for this? It would be helpful for me to fix the issue.

Thanks,
Meghana.
